app/assets/v2/js/pages/bounty_details.js

```diff
@@ -496,25 +496,52 @@ var show_interest_modal = function() {
         modalClass: 'modal add-interest-modal'
       });
 
+      var placeholder = gettext(
+        'What steps will you take to complete this task?\n\n' +
+        'Example:\n' +
+        'I\'d start by implementing Netflify, then I\'d make mockups ' +
+        'for the /blog page and individual post pages and proceed ' +
+        'with building them after getting some feedback from the team.' +
+        'I will start work on this right away and have it ready by the end of the weekend.');
+
+      var textarea = modal.find('textarea');
```

Comment:
This seems to me a very lose query selection. I would like to find by some id, or at least some class, so as to prevent conflicts in future if we plan to add more textareas in the modal.